gravity.sh

```diff
@@ -552,58 +541,35 @@ gravity_SortAndFilterConsolidatedList() {
   if [[ "${haveSourceUrls}" == true ]]; then
```

Comment:
It may be worthwhile to remove the `haveSourceUrls` variable. It seems to indicate if we have any adlists? If we do not have any, then we should exit earlier than this.